.github/workflows/build-nethermind-packages.yml

```diff
@@ -61,40 +53,40 @@ jobs:
         echo "BUILD_TIMESTAMP=$build_timestamp" >> $GITHUB_OUTPUT
         echo "COMMIT_HASH=$commit_hash" >> $GITHUB_OUTPUT
         ./scripts/deployment/build-runner.sh ${{ github.event.inputs.version }} $commit_hash $build_timestamp
-    - name: Building Cli
+    - name: Build Nethermind.Cli
       run: ./nethermind/scripts/deployment/build-cli.sh
-    - name: Building Launcher
+    - name: Build Nethermind.Launcher
       run: ./nethermind/scripts/deployment/build-launcher.sh
-    - name: Building Launcher for arm64
+    - name: Build Nethermind.Launcher for Linux arm64
       run: |
         cd nethermind/
         docker buildx build --platform=linux/arm64 -t tmp-launcher -f Dockerfile.launcher . --load
         docker run --platform=linux/arm64 -v $PWD:/opt/mount --rm tmp-launcher bash -c "cp /nethermind/Nethermind.Launcher /opt/mount/"
-        mv Nethermind.Launcher ${{ env.RELEASE_DIRECTORY }}/${{ env.LIN_ARM64_RELEASE }}/Nethermind.Launcher
-    - name: Archiving packages
+        mv Nethermind.Launcher ${{ env.RELEASE_PATH }}/linux-arm64/Nethermind.Launcher
```

Comment:
Would it be possible to change: `linux-arm64` to a variable if we ever need to change it in future? Same goes for the other names: 

- linux-arm64
- linux-x64
- win-x64
- osx-x64
- osx-arm64

Same question for other files btw, not just this one in particular.

Reply:
I deliberately ditched env vars usage for those, as there is no sense to replace `linux-arm64` with `$LINUX_ARM64` or similar. It's the same string. So, if you would ever want to change it, you would have to change env var name as well. Moreover, `linux-arm64` is not an arbitrary value but a RID of .NET. So its value can be considered a constant.